app/src/templates/guest/welcome.pug

```diff
@@ -3,7 +3,7 @@
     .content(flex, layout="column", layout-align="center center") 
         .logo
         .header selfkey
-        .subheader blockchain identity wallet
+        .subheader blockchain identity wallet member.setup.choose
```

Comment:
why there is added **member.setup.choose** at the end of this line ?

Reply:
It's nothing. I will delete it. Maybe I copy it and paste it by accident.